pkg/kubeadm/master/addons.go

```diff
@@ -223,11 +230,15 @@ func createKubeDNSServiceSpec(s *kubeadmapi.KubeadmConfig) (*api.ServiceSpec, er
 }
 
 func CreateEssentialAddons(s *kubeadmapi.KubeadmConfig, client *clientset.Clientset) error {
-	kubeProxyDaemonSet := NewDaemonSet(kubeProxy, createKubeProxyPodSpec(s))
-	SetMasterTaintTolerations(&kubeProxyDaemonSet.Spec.Template.ObjectMeta)
+	arches := [3]string{"amd64", "arm", "arm64"}
+
+	for _, arch := range arches {
```

Comment:
In other words, please make it multi-arch-ready, but don't no enable deploying for arches other then `runtime.GOARCH` for now.